Classes/Queue/DoctrineQueue.php

```diff
@@ -116,7 +116,7 @@ public function setUp(): void
         $this->connection->exec($createDatabaseStatement);
         try {
             $this->connection->exec("CREATE INDEX state_scheduled ON {$this->connection->quoteIdentifier($this->tableName)} (state, scheduled)");
-        } catch (Exception $e) {
+        } catch (\Doctrine\DBAL\Exception $e) {
```

Comment:
Actually.. I think it's error prone to just silently catch all kinds of exceptions here.
Shouldn't we just catch the specific exception for the mentioned case instead?

Reply:
Following the `exec` method, I found that it throws a `\Doctrine\DBAL\Exception` in it's method

Reply:
It's a `Doctrine\DBAL\Exception\DriverException` for me.. But since that's not really more specific I changed it to the (already imported) `DBALException` now and that works for me